include/fmt/format-inl.h

```diff
@@ -1072,33 +995,46 @@ template <> struct cache_accessor<float> {
         0xb1a2bc2ec5000000, 0xde0b6b3a76400000, 0x8ac7230489e80000,
         0xad78ebc5ac620000, 0xd8d726b7177a8000, 0x878678326eac9000,
         0xa968163f0a57b400, 0xd3c21bcecceda100, 0x84595161401484a0,
-        0xa56fa5b99019a5c8, 0xcecb8f27f4200f3a, 0x813f3978f8940984,
-        0xa18f07d736b90be5, 0xc9f2c9cd04674ede, 0xfc6f7c4045812296,
-        0x9dc5ada82b70b59d, 0xc5371912364ce305, 0xf684df56c3e01bc6,
-        0x9a130b963a6c115c, 0xc097ce7bc90715b3, 0xf0bdc21abb48db20,
-        0x96769950b50d88f4, 0xbc143fa4e250eb31, 0xeb194f8e1ae525fd,
-        0x92efd1b8d0cf37be, 0xb7abc627050305ad, 0xe596b7b0c643c719,
-        0x8f7e32ce7bea5c6f, 0xb35dbf821ae4f38b, 0xe0352f62a19e306e};
+        0xa56fa5b99019a5c8, 0xcecb8f27f4200f3a, 0x813f3978f8940985,
+        0xa18f07d736b90be6, 0xc9f2c9cd04674edf, 0xfc6f7c4045812297,
+        0x9dc5ada82b70b59e, 0xc5371912364ce306, 0xf684df56c3e01bc7,
+        0x9a130b963a6c115d, 0xc097ce7bc90715b4, 0xf0bdc21abb48db21,
+        0x96769950b50d88f5, 0xbc143fa4e250eb32, 0xeb194f8e1ae525fe,
+        0x92efd1b8d0cf37bf, 0xb7abc627050305ae, 0xe596b7b0c643c71a,
+        0x8f7e32ce7bea5c70, 0xb35dbf821ae4f38c, 0xe0352f62a19e306f};
```

Comment:
Why did these change?

Reply:
As I explained in the PR message, it simplifies the task of recovering the full cache from the compressed cache (before the change, entries for positive powers were floor of the true values while entries for negative powers were ceilings of the true values, but now they are all ceilings, so recovery routine does not need to check if the exponent is positive/negative), and also allows replacing integer checks by a single comparison. Detailed explanation of why it's possible might be too long to be written here. I'm currently revising Dragonbox paper with these changes.

But anyway here is a brief explanation. The idea of the new integer check mechanism (which is what Schubfach has been already doing from the beginning) is that when we multiply a number with the approximate power of 5, the error must be smaller than a certain threshold if and only if the true value of the multiplication is an integer. In our case, the minimum amount of error for non-integer product can be shown to be at least 2^-64, while it must be smaller than 2^-64 if the product is an integer. This can be only achieved with ceilings and it doesn't work with floors, hence the change is made.

Reply:
By the way, this is also the core idea of the new implementation of `check_divisibility_and_divide_by_pow10`. In this case as the size of input is small (<= 1000) so I did an exhaustive test to ensure correctness. (https://github.com/jk-jeon/dragonbox/blob/master/subproject/test/source/verify_magic_division.cpp#L24)